importer/tests/test_import.py

```diff
@@ -109,3 +115,106 @@ def test_import_task(self):
             Course.objects.count(),
             original_count + 1,
         )
+
+    @staticmethod
+    def test_import_course_from_path():
+        """
+        Validate parameters and returns fir course from path.
+        """
+        test_path = '/tmp/foo'
+        test_repo_id = 2
+        test_user_id = 42
+        with mock.patch('importer.api.import_course') as mock_import:
+            with mock.patch(
+                'importer.api.import_static_assets'
+            ) as mock_static:
+                with mock.patch('importer.api.XBundle') as mock_bundle:
+                    with mock.patch('importer.api.isdir') as mock_is_dir:
+                        mock_import.return_value = True
+                        mock_is_dir.return_value = True
+                        import_course_from_path(
+                            test_path, test_repo_id, test_user_id
+                        )
+                        mock_import.assert_called_with(
+                            mock_bundle(), test_repo_id, test_user_id
+                        )
+                        mock_static.assert_called_with(
+                            os.path.join(test_path, 'static'), True
+                        )
+
```

Comment:
That is all done in https://github.com/mitodl/lore/blob/feature/upload_assets/importer/api/__init__.py#L28.  The only caller of import_course_from_path I'm aware is that.  We could _ this method in fact